From 8d99949c4c51d95c14fb2b09d18e1cff058f0c17 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Tue, 18 Jul 2017 06:58:16 +0200
Subject: [PATCH 07/18] DOC: update the list of OpenSSL versions in the README

1.1.0 is also supported nowadays. Also mention the best effort support
for derivatives.
(cherry picked from commit 7ab16868bc6e9d5ef879e1046effa035789835cc)

Signed-off-by: Willy Tarreau <w@1wt.eu>
---
 README | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/README b/README
index 839d06ec..8ad70e66 100644
--- a/README
+++ b/README
@@ -113,8 +113,12 @@ build fails due to missing symbols such as deflateInit(), then try again with
 Your are strongly encouraged to always use an up-to-date version of OpenSSL, as
 found on https://www.openssl.org/ as vulnerabilities are occasionally found and
 you don't want them on your systems. HAProxy is known to build correctly on all
-currently supported branches (0.9.8, 1.0.0, 1.0.1 and 1.0.2 at the time of
-writing). Branch 1.0.2 is recommended for the richest features.
+currently supported branches (0.9.8, 1.0.0, 1.0.1, 1.0.2 and 1.1.0 at the time
+of writing). Branch 1.0.2 is currently recommended for the best combination of
+features and stability. Asynchronous engines require OpenSSL 1.1.0 though. It's
+worth mentionning that some OpenSSL derivatives are also reported to work but
+may occasionally break. Patches to fix them are welcome but please read the
+CONTRIBUTING file first.
 
 To link OpenSSL statically against haproxy, build OpenSSL with the no-shared
 keyword and install it to a local directory, so your system is not affected :
-- 
2.13.0

